### DISCUSSION TITLE
Welcome to qttabbar Discussions!

### DISCUSSION THREAD

**Discussion by indiff** [indiff/qttabbar#282] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Andree11gor** [comment]
Cool program, thanks! Question. Is it possible to make a transparent window in the program and adjust the degree of transparency instead of filling it with color? If not, then maybe we should implement such a function in the next version? ;)
